### PR TITLE
Updating CONTRIBUTING.md 

### PR DESCRIPTION
**Pull request by icazevedo**
## What's the purpose of this pull request?
This PR updates the CONTRIBUTING.md guide to not include the commands to release the application, since we have a bot that does that for us now.

I was following this guide and almost re-released the version I merged, so I think it would be good for contributors to have a disclaimer about the bot. Also, I think the command may still be useful, so I left it there with a disclaimer.

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -52,30 +52,27 @@ Example:
 
 1. After acquiring the necessary approval, rebase and squash-merge your PR in master
 
-2. Locally, go to the master branch pull the changes with `git pull origin master`
+2. A bot will commit on master releasing your changes. In the mean time, go grab a coffe ☕
```
Comment (Gmantiqueira) [inline CONTRIBUTING.MD:55]: Typo: `meantime` and `coffee`

File: CONTRIBUTING.MD [hunk 2]
```diff
@@ -52,30 +52,27 @@ Example:
...
 
-3. Clear any artifact and release a clean build with
-```
-git clean -fdx && yarn && lerna run build && lerna publish minor --yes
-```
-
-4. Go grab a coffe ☕
+> If the bot isn't working or you want to release it manually, use the following command to clean any artifact and release a clean build:
+>
+> `git clean -fdx && yarn && lerna run build && lerna publish minor --yes`
```
Comment (Gmantiqueira) [inline CONTRIBUTING.MD:59]: Is it ok to release minor version here?

**Review by tlgimenes — APPROVED** [review]

File: CONTRIBUTING.MD [hunk 3]
```diff
@@ -52,30 +52,27 @@ Example:
...
 ```
 
-7. Rebase with master
+5. Rebase with master
 ```
 git rebase origin/master && git push --force
 ```
 
-8. Wait for the bots to approve the PR.
+6. Wait for the bots to approve the PR.
```
Comment (igorbrasileiro) [inline CONTRIBUTING.MD:76]: I would rewrite it to something like that,
```suggestion
6. Wait for the PR checks to have been passed
```

File: CONTRIBUTING.MD (hunk 2 above)
Comment (tlgimenes) [reply CONTRIBUTING.MD:59]: patch!!

**Review by igorbrasileiro — APPROVED** [review]
lgtm!! Good to see you updating the docs, thank you 😄

File: CONTRIBUTING.MD (hunk 1 above)
Comment (icazevedo) [reply CONTRIBUTING.MD:55]: Oops

Comment (emersonlaurentino) [inline CONTRIBUTING.MD]:
```suggestion
2. A bot will commit on master releasing your changes. In the meantime, go grab a coffee ☕
```

**Review by emersonlaurentino — APPROVED** [review]

**Pull request merged**